src/ProjectSettings.ts

```diff
@@ -90,15 +91,18 @@ export async function selectTemplateFilter(ui: IUserInterface): Promise<Template
     return <TemplateFilter>result;
 }
 
+export function tryGetLanguageSetting(): ProjectLanguage | undefined {
```

Comment:
You have three functions that getConfiguration based off extensionPrefix and the setting name.  Is it time to create a getWorkspaceSetting function?

Reply:
https://github.com/Microsoft/vscode-azurefunctions/pull/177/files#diff-67685aa049a8b0e5dfd64a65893c8717R94 😆

Reply:
Great minds think alike